scripts/vippy/rest.py

```diff
@@ -455,8 +471,10 @@ def make_table(self, data_type):
             tags = data_type.tags
             widths = self.make_widths(self.headers, tags)
             lines = [self.make_divider(widths)]
-            lines.extend(self.make_row(self.headers, widths, separator='='))
+            lines.extend(self.make_row(self.headers, widths, separator="="))
             for tag in tags:
+                if mode == tag.get("skip_on", ""):
```

Comment:
Here is the crucial logic: if the tag contains a line saying "skip_on: csv", will not incorporate the tag into that RST file. This allows us to have some elements (e.g. parent_id) that are exclusive to either CSV or XML.

Reply:
suggtestion, non-blocking

is "skip_on" a name that we choose? i did some googling and such and i think it is, but i may be wrong. if it is a name we choose, how would you feel about using a logically positive identifier rather than a logically negative one? that is to say, instead of `skip_on: xml` we could have `relevant_specification` which could be "csv", "xml", or "both". i find that somewhat easier to read - but that's just me!

am I overthinking this? i might be overthinking this. and it's subjective after all, so feel free to disregard. plus theres the non-trivial downside of needing to add this flag to everything - but i kind of consider that a positive in the long run. food for thought!

Reply:
I will think about this. I agree about reading affirmative vs negatives.